powerVCF.psm1

```diff
@@ -2119,6 +2119,53 @@ Function Get-VCFUpgradables {
 }
 
 ######### End Get Upgradable Operations ##########
+######### Start Upload Bundle Operations ##########
+
+Function Start-UploadVCFBundle {
+<#
+  .SYNOPSIS
+  Starts upload of bundle to SDDC Manager
+
+  .DESCRIPTION
+  The Start-UploadVCFBundle cmdlet starts upload of bundle(s) to SDDC Manager
+
+  .EXAMPLE
+  PS C:\> Start-UploadVCFBundle -json .\Bundle\bundlespec.json
+  This example invokes the upload of a bundle onto SDDC Manager
+#>
+
+  Param (
+    [Parameter (Mandatory=$true)]
+      [ValidateNotNullOrEmpty()]
+      [string]$json
+  )
+
+  $headers = @{"Accept" = "application/json"}
```

Comment:
Check with Gary as he has made a change around headers to reduce code

Reply:
**Replace:**
$headers = @{"Accept" = "application/json"}
$headers.Add("Authorization", "Basic $base64AuthInfo")

**With the new function, using the following line:**
createHeader # Calls Function createHeader to set Accept & Authorization

Reply:
Fixed